### ISSUE TITLE
Prisma CLI keeps downloading files that should only be downloaded once

### ISSUE THREAD

**Issue by janpio** [denoland/deno#20098]
On installation Prisma CLI (Npm package `prisma`) downloads some files (some binaries and some Node-API based libraries) from a CDN (`binaries.prisma.sh`) and puts them in `node_modules` for later use. `prisma -v` then outputs the path to these and asks the binaries and Node-API library for their version to output.

In my setup on Windows, each time I run `prisma -v` it downloads these files again:
<img width="632" alt="image" src="https://github.com/denoland/deno/assets/183673/b7c7a61e-cb35-41a3-a9d6-fc68c5a35e82">

Reproduction might be as simple as creating a new folder, then running `deno run -A npm:prisma init` and then `deno run -A npm:prisma -v` to observe it downloading the file again each time you run the command.

(When running `prisma -v` I also get this error message regularly when the CLI tries to ask one of the binaries for its version: https://github.com/denoland/deno/issues/19544)

```
deno --version
deno 1.36.0 (release, x86_64-pc-windows-msvc)
v8 11.6.189.12
typescript 5.1.6
```

**Comment by janpio** [comment]
(Sometimes additionally to downloading, also this error is thrown first: https://github.com/denoland/deno/issues/19544#issuecomment-1670358903)

**Comment by kt3k** [comment]
We assume this has been fixed in #21297. Could you check the above again with Deno v1.38.3?

**Comment by janpio** [comment]
Looks good, it download the engines once and then not any more - as intended. Thanks!
